packages/core/src/client/models.ts

```diff
@@ -1,6 +1,20 @@
 import { SitecoreConfig } from '../config';
-import { SiteInfo } from '../site';
+import { GraphQLEditingService } from '../editing/graphql-editing-service';
+import { RestComponentLayoutService } from '../editing/rest-component-layout-service';
+import { GraphQLDictionaryService } from '../i18n/graphql-dictionary-service';
+import { GraphQLLayoutService } from '../layout/graphql-layout-service';
+import { GraphQLErrorPagesService, GraphQLSitePathService, SiteInfo, SiteResolver } from '../site';
 
+/**
+ * Init options for Sitecore Client that allows you to override services too
+ */
 export type SitecoreClientInit = Omit<SitecoreConfig, 'multisite' | 'redirects' | 'personalize'> & {
   sites: SiteInfo[];
+  layoutService?: GraphQLLayoutService;
```

Comment:
should we combine all the services under single property, like `services`, or `custom` or `overrides`, to not mix it with 'real' settings?
I don't have a strong opinion about that, what do you think about it?